src/main/java/com/codeofus/rent_a_park/models/Person.java

```diff
@@ -0,0 +1,31 @@
+package com.codeofus.rent_a_park.models;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Entity
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Person {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
```

Comment:
Why sequence and not identity? There's a cool optimisation possible when using IDENTITY, would it work in a case we have more than one instance of this app deployed